jvcl/run/JvRollOut.pas

```diff
@@ -993,39 +993,45 @@ procedure TJvCustomRollOut.SetExpandedSize(const Value: integer);
      ((FPlacement = plLeft) and (FAWidth = Value)) then
     Exit;
 
-  if FPlacement = plTop then
-    FAHeight := Value
-  else
-    FAWidth := Value;
+  FTopForm.DisableAlign;
 
-  if not FCollapsed then
-  begin
-    // The top form is assigned so set the width and height of this form
-    if Parent = FTopForm then
+  try
+    if FPlacement = plTop then
+      FAHeight := Value
+    else
+      FAWidth := Value;
+
+    if not FCollapsed then
     begin
-      FTopForm.DisableAlign;
-      if FPlacement = plTop then
+      // The top form is assigned so set the width and height of this form
+      if Parent = FTopForm then
       begin
-        FTopForm.Height := FAHeight;
-        FOldWidthHeight.Y := FAHeight;
-      end
-      else
-      begin
-        FTopForm.Width := FAWidth;
-        FOldWidthHeight.X := FAWidth;
+        FTopForm.DisableAlign;
+        if FPlacement = plTop then
+        begin
+          FTopForm.Height := FAHeight;
+          FOldWidthHeight.Y := FAHeight;
+        end
+        else
+        begin
+          FTopForm.Width := FAWidth;
+          FOldWidthHeight.X := FAWidth;
+        end;
+        FTopForm.EnableAlign;
```

Comment:
Now you have this ``EnableAlign`` and the one some lines later (around 1034) in your ``finally`` block.

Reply:
Ok, I fixed that now. Redoing the pr was obviously done a bit in a hurry. Sorry. I really hope it is good enough for getting merged in now!